tests/func/test_add.py

```diff
@@ -44,7 +44,12 @@ def test_add(tmp_dir, dvc):
     assert len(stage.deps) == 0
     assert stage.cmd is None
     assert stage.outs[0].info["md5"] == md5
-    assert stage.md5 == "ee343f2482f53efffc109be83cc976ac"
+    assert stage.md5 == "411854a9fc55b3cebda28704e41b23b3"
```

Comment:
Yep, have to break md5 again. But no worries, we'll get rid of it for good for `dvc add`ed files soon.